docker/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.6' # compatible with ubuntu bionic
```

Comment:
I had to downgrade this to version 2.0 to run on Ubuntu18.04. This version was also not compatible with Ubuntu16.04. 
I downloaded the most recent docker-compose and faced this error.

Reply:
Just to be sure can you tell me what you get when you run `docker-compose --version`

Reply:
for reference im using docker-compose version 1.27.4, build 40524192

Reply:
@MewtR docker-compose version 1.27.4, build unknown is what I have setup

Reply:
Turns out using pip3 install docker-compose causes an unknown build to be set